### ISSUE TITLE
Updated to 6.2.2.GA

### ISSUE THREAD

**Issue by securit** [yozef/FusedLocation#6]
Any chance of updating to 6.2.2GA?  I see the google play services (com.google.android.gms.version) is registered at 9683000 whereas the current release is 10298000. Why did you use a fixed version rather than @integer/google_play_services_version ?

**Comment by yozef** [comment]
I haven’t tried it on the latest Ti SDK, do you have any suggestions on not setting a fixed version for Google Play Services, so it could be used with to 6.2.2 and beyond?

**Comment by securit** [comment]
Thats what the @integer/google_play_services_version does? You set that in your timodule.xml file I think

**Comment by yozef** [comment]
It requires also changes to the jar files since they're from the 9683000 version. Will update once I remember how I did it the first time :-p

**Comment by securit** [comment]
Good luck with that. I've been trying to figure out how to do this with Hyperloop instead... Not so easy.

**Comment by securit** [comment]
this may help ;) https://medium.com/all-titanium/deal-with-titanium-modules-and-its-missing-support-for-android-multidex-546de5486d13

**Comment by yozef** [comment]
Yep that was it! Feel free to make a pull request if you manage to get it working on Ti 6.2.2

**Comment by DouglasHennrich** [comment]
[google-play-services.jar.zip](https://github.com/yozef/FusedLocation/files/1427236/google-play-services.jar.zip)

Version: `10084000` must change on `@integer/google_play_services_version`
it works with `ti.map` as well... just need to replace all files from `lib` for this `google-play-services` and change it's `@integer/google_play_services_version`

**Comment by yozef** [comment]
Thanks @DouglasHennrich I'm currently out of office till Thursday, I will check it out, and update the code if all is well & works. Thanks for the update & help :)

**Comment by joshualambert** [comment]
Awesome @DouglasHennrich -- I'll give this a shot today also and see if it works on my app. You rock.

**Comment by securit** [comment]
@DouglasHennrich do you have a version that doesn't conflict with Ti.Maps? I've tried configuring a proguard.conf to strip out the conflicts, but I'm still struggling a bit with it.

**Comment by DouglasHennrich** [comment]
as I said, just put this file into map and fusion lib and it will work

**Comment by joshualambert** [comment]
I swapped the libraries, still getting Dexer errors:
```[ERROR] Failed to run dexer:
[ERROR]
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/ItemListIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/NoteIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/ReserveIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/SearchIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$1;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$Info;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$zza;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/zza;
[ERROR]
[ERROR] UNEXPECTED TOP-LEVEL EXCEPTION:
[ERROR] java.lang.RuntimeException: Translation has been interrupted
[ERROR]         at com.android.dx.command.dexer.Main.processAllFiles(Main.java:615)
[ERROR]         at com.android.dx.command.dexer.Main.runMultiDex(Main.java:368)
[ERROR]         at com.android.dx.command.dexer.Main.runDx(Main.java:289)
[ERROR]         at com.android.dx.command.dexer.Main.main(Main.java:247)
[ERROR]         at com.android.dx.command.Main.main(Main.java:94)
[ERROR] Caused by: java.lang.InterruptedException: Too many errors
[ERROR]         at com.android.dx.command.dexer.Main.processAllFiles(Main.java:607)
[ERROR]         ... 4 more```

**Comment by joshualambert** [comment]
Placed the new JAR inside of libs, removed the other 3.

**Comment by joshualambert** [comment]
<img width="601" alt="screen shot 2017-10-31 at 8 37 32 am" src="https://user-images.githubusercontent.com/1421842/32226531-c1e36b22-be16-11e7-84bb-fffc8d223940.png">

**Comment by DouglasHennrich** [comment]
![captura de tela 2017-10-31 as 11 40 53](https://user-images.githubusercontent.com/10107024/32226687-64f4b870-be30-11e7-895e-dac3df33f7f4.png)
![captura de tela 2017-10-31 as 11 41 00](https://user-images.githubusercontent.com/10107024/32226688-652b42dc-be30-11e7-98b2-ee39016243ff.png)

**Comment by DouglasHennrich** [comment]
@joshualambert also... there's another android module that you're using ?

**Comment by joshualambert** [comment]
@DouglasHennrich What do you mean by "another android module"? I've got a number of Android modules in this app: `ti.imagefactory`, `ti.map`, `ti.draggable`, `ti.flurry`,  `facebook`, and possibly `FusedLocation` if I can get this to work.

**Comment by DouglasHennrich** [comment]
I guess ti.flurry use google play services as well... try to remove that module and see if it builds.
You need to do like I did on both modules

**Comment by joshualambert** [comment]
Totally removed Flurry. Same issue:```
 [ERROR] Failed to run dexer:
[ERROR]
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/ItemListIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/NoteIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/ReserveIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/actions/SearchIntents;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$1;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$Info;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient$zza;
[ERROR] Uncaught translation error: java.lang.IllegalArgumentException: already added: Lcom/google/android/gms/ads/identifier/zza;
[ERROR]
[ERROR] UNEXPECTED TOP-LEVEL EXCEPTION:
[ERROR] java.lang.RuntimeException: Translation has been interrupted
[ERROR]         at com.android.dx.command.dexer.Main.processAllFiles(Main.java:615)
[ERROR]         at com.android.dx.command.dexer.Main.runMultiDex(Main.java:368)
[ERROR]         at com.android.dx.command.dexer.Main.runDx(Main.java:289)
[ERROR]         at com.android.dx.command.dexer.Main.main(Main.java:247)
[ERROR]         at com.android.dx.command.Main.main(Main.java:94)
[ERROR] Caused by: java.lang.InterruptedException: Too many errors
[ERROR]         at com.android.dx.command.dexer.Main.processAllFiles(Main.java:607)
[ERROR]         ... 4 more
```

**Comment by joshualambert** [comment]
Confirmed this is happening on both 6.1.0 and 6.2.0.

**Comment by DouglasHennrich** [comment]
show me your map and fused folder under lib

**Comment by securit** [comment]
FYI - "Google Play Services implementation has been improved by removing the library from Appcelerator-provided modules and providing a ti.playservices module for a single and easier implementation. No longer will users be required to keep modules that use Google Play Services in sync to avoid dexer errors. We are including our Google Play Services module and providing updated versions of our other modules such as ti.map, ti.cloudpush and ti.admob. Users simply need to include ti.playservices module as a dependency in their own module’s timodule.xml to make use of this new design. For more info, see an example on Github." Currently in the 7.0 Beta. Secondly Fused Location is now scheduled to be released in 7.1 SDK.

**Comment by DouglasHennrich** [comment]
oh god... 7.1 🙄

**Comment by securit** [comment]
Yup... 7.0 RC is now out.... but the fuseLocation provider availability has been updatged to:
Fix Version/s |   | Release 7.1.0 [ 19957 ]
